app/src/components/dataset/EditLLMDatasetEvaluatorSlideover.tsx

```diff
@@ -202,13 +203,15 @@ const EditEvaluatorDialog = ({
       evaluator: {
         name: datasetEvaluator.displayName ?? "",
         description: datasetEvaluator.evaluator.description ?? "",
+        kind: datasetEvaluator.evaluator.kind,
+        isBuiltin: false,
       },
-      choiceConfig: {
+      outputConfig: {
         name: datasetEvaluator.evaluator.outputConfig?.name ?? "",
         optimizationDirection:
           datasetEvaluator.evaluator.outputConfig?.optimizationDirection ??
           "MAXIMIZE",
-        choices: datasetEvaluator.evaluator.outputConfig?.values.map(
+        values: datasetEvaluator.evaluator.outputConfig?.values.map(
```

Comment:
we should strongly type this huh. I can imagine not  all of it being a list.

Reply:
I think it is strongly typed, outputConfig just may not exist if the ... on LLMEvaluator fragment doesn't resolve